UI Test Runner/UITests/FloorFilterTests.swift

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+
+final class FloorFilterTests: XCTestCase {
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+    }
+    
+    /// Test general usage of the Floor Filter component.
+    func testFloorFilter() throws {
+        let app = XCUIApplication()
+        app.launch()
+        
+        // Open the Floor Filter component test view.
+        app.buttons["Floor Filter Tests"].tap()
+        
+        // Wait for floor aware data to load and then open the filter.
+        let filterButton = app.buttons["Business"]
+        _ = filterButton.waitForExistence(timeout: 5)
```

Comment:
Seeing this value discarded made me think of what is mostly missing in these tests: assertion. While interacting with a non-existent element (like tapping a missing button) will fail the test, the failure message isn't pretty. In Apple's antiquated [User Interface Testing](https://developer.apple.com/library/archive/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/09-ui_testing.html), they give a good pattern for UI tests:

> The general pattern of a UI test for correctness is as follows:
> 
> - Use an XCUIElementQuery to find an XCUIElement.
> - Synthesize an event and send it to the XCUIElement.
> - Use an assertion to compare the state of the XCUIElement against an expected reference state.

And if you look at some of Apple's recent UI testing example code, such as in WWDC 2021's [Explore Digital Crown, Trackpad, and iPad pointer automation](https://developer.apple.com/videos/play/wwdc2021/10208/), you'll see that they follow that pattern:

```swift
func testHorizontalScrollRevealsSidebar() throws {
    let app = XCUIApplication()
    app.launch()
    
    let sidebar = app.tables["Sidebar"]
    
    // Make sure sidebar menu is not present initially.
    XCTAssertFalse(sidebar.exists, "Sidebar should not be present initially")
    
    // Swipe horizontally to reveal sidebar.
    app.staticTexts["Select a smoothie"].scroll(byDeltaX: -200, deltaY: 0)
    
    // Verify sidebar is now present.
    XCTAssertTrue(sidebar.waitForExistence(timeout: 5),
                  "Sidebar did not appear within 5 second timeout")
}
```

All that to say, you might want to add some more assertions, including here.